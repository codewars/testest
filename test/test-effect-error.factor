! Copyright 2024 nomennescio

USING: tools.testest kernel math ;
IN: tests

: wont-infer ( -- ) 0 ;

: run-tests ( -- )

"Compilation error" describe#{
  "Compilation error in user code" it#{
     <{ wont-infer -> }>
  }#
}#
;

MAIN: run-tests