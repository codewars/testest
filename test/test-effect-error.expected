
<DESCRIBE::>Compilation error

<IT::>Compilation error in user code

==== test-effect-error.factor

test-effect-error.factor: 6

Asset: wont-infer

Stack effect declaration is wrong
inferred ( -- x )
declared ( -- )

<COMPLETEDIN::>0.345100 ms

<COMPLETEDIN::>0.516400 ms
